src/requests/getImages.js

```diff
@@ -0,0 +1,26 @@
+ import axios from "axios";
+
+const getImages = (query) => {
+  if (!query) {
+    return Promise.resolve([]);
+  } else {
+    const API_URL = `https://images-api.nasa.gov/search?media_type=image&year_start=1920&year_end=2023&q=`;
+    const API_KEY = 'your_api_key_here';
+
+    return axios
+      .get(`${API_URL}${query}`, {
+        headers: { 'Authorization': API_KEY }
+      })
+      .then((response) => {
+        const imageResults = response.data.collection.items;
+        const parsedImages = imageResults.filter(item => item.data[0].media_type === 'image');
+        const images = parsedImages.map(item => item.links[0].href);
+        return images;
+      })
+      .catch((err) => {
+        console.log(err);
+      });
```

Comment:
Instead of logging the error to console using `console.log`, you can use `console.error` instead

Reply:
You should also add the line `return Promise.resolve([]);` after your `console.log`, as if an error is thrown your app would break, as it sets the value of `searchResults` to `undefined`, which would break the `map` in `SearchResults`.